src/jquery.valueview.valueview.js

```diff
@@ -183,7 +183,7 @@ $.widget( 'valueview.valueview', PARENT, {
 		value: null,
 		language: null,
 		autoStartEditing: false,
-		parseDelay: 300,
+		parseDelay: 500,
```

Comment:
Changing the delay is not directly necessary to fix the problem, am I right? I would not change it in that case. AFAIK we have this limit defined in several other places too, so should be carefully evaluated.